src/oplog/log.js

```diff
@@ -405,31 +423,6 @@ const Log = async (identity, { logId, logHeads, access, entryStorage, headsStora
     await _entries.close()
   }
 
-  /**
-   * TODO
-   * Get references at every pow2 distance
-   * If pointer count is 4, returns 2
-   * If pointer count is 8, returns 3 references
-   * If pointer count is 512, returns 9 references
-   * If pointer count is 2048, returns 11 references
-   */
-  const getReferences = async (pointerCount = 1) => {
-    let nextPointerDistance = 2
-    let distance = 0
-    const refs = []
-    const shouldStopFn = () => distance >= pointerCount
-    for await (const entry of traverse(null, shouldStopFn)) {
-      distance++
-      if (distance === nextPointerDistance) {
-        if (entry.hash) {
-          refs.push(entry.hash)
-        }
-        nextPointerDistance *= 2
-      }
-    }
-    return refs
-  }
-
```

Comment:
The `refs` are now simply the next `n` entries, so we can remove the pow2 counting.